license_check.py

```diff
@@ -105,7 +106,6 @@ def main():
     spdx_expr = args.license
     if not spdx_expr:
         spdx_expr = 'Apache-2.0'
-        eprint(f'No SPDX expression supplied. Using {spdx_expr}.')
```

Comment:
This seems to be unrelated to what the PR/commit is proposing. Should be in a different commit at least.